src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/scene_settings_widget.py

```diff
@@ -32,6 +32,22 @@
 COMBO_DEFAULT_ALL_RENDERABLE_CAMERAS = "All Renderable Cameras"
 COMBO_DEFAULT_ALL_RENDERABLE_LAYERS = "All Renderable Layers"
 
+_BLENDER_OUTPUT_FORMATS: dict[str, str] = {
```

Comment:
Does this need to be private?

Reply:
I don't think it *needs* to. I was following Keyshot which has this dictionary private but I can make it public if that might help future work.

Reply:
Oh, that should be fine I believe. I'll make a separate function in `scene_settings_widget.py` for the validation

Reply:
Chatted offline with Mia and the output format handling will be a separate PR